src/exo/API.py

```diff
@@ -200,14 +200,30 @@ def is_instr(self):
     def get_instr(self):
         return self._loopir_proc.instr
 
+    def find_cursor(self, pattern):
+        cursors = match_cursors(self, pattern, call_depth=1)
+        assert isinstance(cursors, list)
+        if not cursors:
+            raise SchedulingError('failed to find statement', pattern=pattern)
```

Comment:
Should say `failed to find cursor` or `failed to find matches` not statement specifically

Reply:
Changed to say "matches" instead of "statement".